client/apps/packages/components/packages-list/index.js

```diff
@@ -60,10 +60,14 @@ const PackagesList = ( { packages, dimensionUnit, editable, selected, serviceId,
 	return (
 		<FormFieldset className="packages-list">
 			<div className="packages-list__header">
-				<FormLegend className="packages-list__column-actions" />
+				{ editable ? null : <FormLegend className="packages-list__column-actions" /> }
 				<FormLegend className="packages-list__column-type">{ __( 'Type' ) }</FormLegend>
 				<FormLegend className="packages-list__column-name">{ __( 'Name' ) }</FormLegend>
-				<FormLegend className="packages-list__column-dimensions">{ __( 'Dimensions (L x W x H)' ) }</FormLegend>
+				<FormLegend className="packages-list__column-dimensions">
+					{ __( 'Dimensions' ) }{ ' ' }
```

Comment:
It was inside its own `<span>` but that `<span>` had no associated CSS, so it was pointless.